tvm/version_manager/domain/version_manager_repository.py

```diff
@@ -39,3 +39,11 @@ def use_version(self, version: TutorVersion) -> None:
     @abstractmethod
     def version_is_installed(version: str) -> None:
         pass
+
+    @abstractmethod
+    def install_plugin(self, version: TutorVersion) -> None:
```

Comment:
This is wrong, install plugin and uninstall plugin doesn't need version, please fix it.